src/test/components/Timeline.test.js

```diff
@@ -276,28 +276,6 @@ describe('Timeline multiple thread selection', function () {
     ]);
   });
 
-  it('will not select a track through a random keypress for a global track', function () {
```

Comment:
After the change, these tests were really testing that the test utility wouldn't send the event... not useful :)